bake/entitlements_test.go

```diff
@@ -343,6 +343,31 @@ func TestValidateEntitlements(t *testing.T) {
 			conf: EntitlementConf{
 				FSRead: []string{"/"},
 			},
+			expected: EntitlementConf{
+				FSRead: func() []string {
+					// on windows root (/) is only allowed if it is the same volume as wd
+					if filepath.VolumeName(wd) == filepath.VolumeName(escapeLink) {
+						return nil
+					}
+					// if not, then escapeLink is not allowed
+					p, err := evaluateToExistingPath(escapeLink)
+					require.NoError(t, err)
+					return []string{p}
+				}(),
+			},
```

Comment:
Revert the change to allow any volumes for `/` on windows and handle that case in our tests instead.